### PR TITLE
{compiler}[dummy/] PGI v17.10

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2072.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/976507648a8a3a3f0838da2322542865 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2498.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/5357838009363d9007434cf98adf43cb for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 2 (1 easyconfigs in this PR)
rescomp2 - Linux centos 6.8, Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz, Python 2.7.11
See https://gist.github.com/20f8b5c7d98c7f862fd8859cc7f67ea2 for a full test report.

**Comment by boegel** [comment]
@verdurin Make sure you use the `numactl` easyconfig from `develop`, to deal with change in download URL.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 1 out of 2 (1 easyconfigs in this PR)
rescomp2 - Linux centos 6.8, Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz, Python 2.7.11
See https://gist.github.com/5cae1a8e76da9e7d0ef02508c82a8f1c for a full test report.

File: easybuild/easyconfigs/p/PGI/PGI-17.10-GCC-6.4.0-2.28.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name = 'PGI'
+version = '17.10'
+
+homepage = 'http://www.pgroup.com/'
+description = "C, C++ and Fortran compilers from The Portland Group - PGI"
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+sources = ['pgilinux-20%(version_major)s-%(version_major)s%(version_minor)s-x86_64.tar.gz']
```
Comment (verdurin) [inline easybuild/easyconfigs/p/PGI/PGI-17.10-GCC-6.4.0-2.28.eb]: @boegel Worth adding a note here about the need to download manually? I know it's implicit owing to the lack of a `sources_url` but...

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
rescomp2 - Linux centos 6.8, Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz, Python 2.7.11
See https://gist.github.com/033dbc502f1384c23583c7af4776409a for a full test report.

**Comment by boegel** [comment]
@verdurin Do you have a PGI license you can use to test this?

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**